tests/invalid/AddressExpression_Invalid_1.sysout

```diff
@@ -0,0 +1,5 @@
+../../tests/invalid/AddressExpression_Invalid_1.whiley:10: unable to resolve name (overloaded(...) is ambiguous
+	found: AddressExpression_Invalid_1:overloaded : function(int)->(int)
+	found: AddressExpression_Invalid_1:overloaded : function(bool)->(bool))
```

Comment:
oops, that sysout fails on travis because it's not really deterministic. The order of both found functions is not guaranteed to be the same as given here.

Shall I remove this test or mark it as ignored?